src/Core/View.js

```diff
@@ -340,6 +386,21 @@ View.prototype.getLayers = function getLayers(filter) {
     return result;
 };
 
+/**
+ * Gets the layer by identifier.
+ *
+ * @param {String}  layerId  The layer identifier
+ * @return {Layer}  The layer by identifier.
+ */
+
+View.prototype.getLayerById = function getLayers(layerId) {
```

Comment:
`function getLayerById` instead.

Also we need to change (in next PR) everywhere this method could be used.

Reply:
done